install

```diff
@@ -205,14 +203,13 @@ if File.directory? HOMEBREW_PREFIX
   sudo "/usr/bin/chgrp", "admin", *chgrps unless chgrps.empty?
 else
   sudo "/bin/mkdir", "-p", HOMEBREW_PREFIX
-  sudo "/bin/chmod", "g+rwx", HOMEBREW_PREFIX
-  # the group is set to wheel by default for some reason
-  sudo "/usr/sbin/chown", "#{ENV['USER']}:admin", HOMEBREW_PREFIX
+  sudo "/usr/sbin/chown", "root:wheel", HOMEBREW_PREFIX
 end
```

Comment:
@tdsmith Yep. Basically: that's the default permissions so if this directory doesn't exist, let's restore them to avoid any weird changes in future.